fatiando/seismic/conv.py

```diff
@@ -0,0 +1,126 @@
+"""
+
+"""
+import numpy as np
+from scipy import interpolate  # linear interpolation of velocity/density
+
+
+def seismic_convolutional_model(n_samples, n_traces, model, f, dz=1.,
+                                dt=2.e-3, rho=1.):
+    """
+    Calculate the synthetic seismogram of a geological depth model, Vp is
+    mandatory while density is optional. The given model in a matrix form is
+    considered a mesh of square cells.
+
+    .. warning::
+
+        Since the relative difference between the model is the important, being
+        consistent with the units chosen for the parameters is the only
+        requirement, whatever the units. Good reference is available at
+        http://dx.doi.org/10.1190/1.9781560801580.ch2
+
+    Parameters:
+
+    * n_samples, n_traces: integers
+        The vertical and horizontal grid dimensions
+    * model : 2D-array
+        Vp values
+    * f : float
+        Dominant frequency of the ricker wavelet
```

Comment:
Could you take the wavelet as input? Do people actually use other wavelets in seismics?